### DISCUSSION TITLE
GripperOpen

### DISCUSSION THREAD

**Discussion by Qiandao-Liu** [empriselab/RCareWorld#56] [Help - Programming] [answered]
hi I was tring to use GripperOpen() and GripperClose() function, but gripper in unity is not opened, could you explain more on how to use these functions? Thanks!

**Comment by YoruCathy** [comment, marked as answer]
Make sure you are controlling the robot rather than the mobile base. If you did that correctly, see if you can get the sponge grasped and dropped using the two functions. The robot is supposed to grasp the sponge if you move it close to the sponge and do Gripper Close.
